spec.bs

```diff
@@ -2233,6 +2244,27 @@ To <dfn>encode trusted signals keys</dfn> given an [=ordered set=] of [=strings=
     encodes it to "%20".
 1. Return |list|.
 
+</div>
+
+<div algorithm>
+
+To <dfn>append to a bidding signals per-interest group data map</dfn> given an [=ordered map=]
+|sourceMap|, an [=ordered set=] |igNames|, and an [=ordered map=] |destinationMap|:
+1. [=map/For each=] |sourceKey| → |sourceValue| of |sourceMap|:
+  1. If |igNames| does not [=set/contain=] |sourceKey|, [=iteration/continue=].
+  1. Let |destinationValue| be an [=ordered map=] whose keys are per-interest group property string
+    name and whose values are the values of those properties.

```

Comment:
What's the data type of "the values of those properties"?
```suggestion
  1. Let |destinationValue| be an [=ordered map=] whose [=map/keys=] are per-interest group property name [=strings=]
    and whose [=values=] are the values of those properties.
```

Reply:
It isn't consistent -- for now, it's just the numeric value for updateIfOlderThanMs, but when we spec the priorityVector, that will have a dictionary, see [0].

[0] https://github.com/WICG/turtledove/blob/main/FLEDGE.md#31-fetching-real-time-data-from-a-trusted-server:~:text=%27priorityVector%27%3A%20%7B%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%27signal1%27%3A%20number%2C%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20%27signal2%27%3A%20number%2C%0A%20%20%20%20%20%20%20%20%20%20%20%20%20%20...%7D

Reply:
ok, we still need to define it. We just need to define it as double for now, and extend it to include more types when needed.

Reply:
Maybe I should make a struct for it then? Then I could just add relevant members to the struct later.

Reply:
I made this a struct -- I think that makes it more clear.